packages/cssnano-preset-default/package.json

```diff
@@ -24,6 +24,7 @@
     "postcss-discard-duplicates": "^4.0.2",
     "postcss-discard-empty": "^4.0.1",
     "postcss-discard-overridden": "^4.0.1",
+    "postcss-lowercase-text": "*",
```

Comment:
Use `1.0.0` verison

Reply:
I didnt publish v`1` . so it will through error if we build it locally. I will release it manually after merging of this.